app/routes/_index/styles.module.css

```diff
@@ -16,13 +8,13 @@ body {
   padding: 1rem;
 }
 
-h1,
-p {
+.h1,
+.p {
```

Comment:
Hm, question about this.  Originally, this style would apply to ALL `h1` and `p` tags, right?  But now it's only those with the given class.  Wouldn't this be better to just apply whatever Module Class name is generated to the outer most component in the `render` function, and still get these applied?

Reply:
Or maybe use the `:global` exception? https://github.com/css-modules/css-modules#exceptions

Reply:
Yeah I'm fine with using more semantic class names, I was debating that, good call out.